frontend/src/App.vue

```diff
@@ -103,6 +104,13 @@ export default {
   },
   methods: {
     ...mapActions("user", ["signOut"])
+  },
+  watch: {
+    '$route' (to) {
+      if (to.path.includes("/sound-editing/upload")) {
+        this.showHeader = false;
+      }
+    }
```

Comment:
I don't like this approach.
Can't we use something like [this](https://forum.vuejs.org/t/multiple-master-page-in-vue-router/20270) or [this](https://stackoverflow.com/q/37838173)?